_data/simple-icons.json

```
@@ -4331,6 +4331,11 @@
             "source": "https://github.com/gruntjs/gruntjs.com/blob/70f43898d9ce8e6cc862ad72bf8a7aee5ca199a9/src/media/grunt-logo-no-wordmark.svg",
             "guidelines": "https://github.com/gruntjs/grunt-docs/blob/main/Grunt-Brand-Guide.md"
         },
+        {
+            "title": "GTK",
+            "hex": "7FE719",
+            "source": "https://upload.wikimedia.org/wikipedia/commons/7/71/GTK_logo.svg"
```

Please use the Wikipedia link to the logo file's _page_ (see [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines)) and add the link to the brand guidelines:

```suggestion
            "source": "https://commons.wikimedia.org/wiki/File:GTK_logo.svg",
            "guidelines": "https://foundation.gnome.org/logo-and-trademarks/"
```

---

Thanks, to me looks good! 🙌

---

This suggested change is still pending. If you agree, could you please add a commit?